src/adapter/src/catalog/builtin.rs

```diff
@@ -2607,6 +2607,24 @@ FROM parent_addrs AS pa
         AND pa.worker_id = oa.worker_id",
 };
 
+pub const MZ_DATAFLOW_ARRANGEMENT_SIZES: BuiltinView = BuiltinView {
+    name: "mz_dataflow_arrangement_sizes",
+    schema: MZ_INTERNAL_SCHEMA,
+    sql: "CREATE VIEW mz_internal.mz_dataflow_arrangement_sizes AS
+SELECT mdo.id, mi.name, sum(mas.records) AS records, sum(mas.batches) AS batches
+FROM
+    mz_internal.mz_dataflow_operators AS mdo
+        JOIN
+            mz_internal.mz_arrangement_sizes AS mas
+            ON mdo.id = mas.operator_id
+        JOIN mz_internal.mz_dataflow_addresses AS mda ON mda.id = mdo.id
+        JOIN
+            mz_internal.mz_compute_exports AS mce
+            ON mce.dataflow_id = mda.address[1]
+        JOIN mz_indexes AS mi ON mi.id = mce.export_id
```

Comment:
Why is that not querying `mz_objects`? As is, we are ignoring MV and subscribe dataflows.

Reply:
Yes, that's a good point.